src/components/Authentication/ForgotPasswordForm.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardDescription, CardFooter, CardHeader, CardTitle } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { useForm } from "react-hook-form";
+import { useState } from "react";
+import * as z from "zod";
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from "../ui/form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import toast from "react-hot-toast";
+
+const formSchema = z.object({
+  email: z.string().email({ message: "Email must be a valid email address" }),
+});
+export type ForgotPassData = z.infer<typeof formSchema>;
```

Comment:
Got plan to use this type elsewhere? Else, remove `export`.
Prefix any custom types with T, custom interfaces with I (capital i).

*Example: `TForgotPasswordData`, `IProps`*.